ui-components/src/blocks/ActionBlock.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react"
+import "./ActionBlock.scss"
+
+export enum ActionBlockLayout {
+  block = "block",
+  inline = "inline",
+}
+
+export enum ActionBlockBackground {
+  none = "none",
+  primaryLighter = "primary-lighter",
+  primaryDarker = "primary-darker",
+}
+interface ActionBlockProps {
+  actions: React.ReactNode[]
+  background?: string
+  header: string
+  icon?: React.ReactNode
+  layout?: ActionBlockLayout
+  subheader?: string
+}
+const ActionBlock = ({
+  actions,
+  background = ActionBlockBackground.none,
+  header,
+  icon,
+  layout = ActionBlockLayout.block,
+  subheader,
+}: ActionBlockProps) => {
+  const actionBlockClasses = ["action-block"]
+  if (background) actionBlockClasses.push(background)
+  layout === "block"
+    ? actionBlockClasses.push("action-block__block")
+    : actionBlockClasses.push("action-block__inline")
```

Comment:
This is completely my preference, and I think we should probably add a lint rule for this, but using ternaries for operations like this is harder to read than a simple if else.